.github/workflows/portable-rustls-ci-build.yml

```
@@ -111,10 +115,18 @@ jobs:
         env:
           RUSTFLAGS: --cfg portable_atomic_unstable_coerce_unsized --cfg unstable_portable_atomic_arc
 
-      - name: cargo test (debug; all features)
+      - name: cargo test (debug; unstable-api-all [should cover most if not all non-unsafe features])
+        # QUICK WORKAROUND TO AVOID TEST ISSUE WITH RUST NIGHTLY FROM MAY 2024
```

extremely minor - ???

```suggestion
        # QUICK WORKAROUND to avoid test issue with Rust nightly-2024-05-...
```